package.json

```diff
@@ -22,11 +22,13 @@
   "author": "Mozilla (https://mozilla.org/)",
   "license": "MPL-2.0",
   "dependencies": {
+    "adr-log": "^2.1.1",
     "chalk": "1.1.1",
     "diffparser": "^2.0.1",
     "fxa-dev-launcher": "vladikoff/fxa-dev-launcher",
     "internal-ip": "1.2.0",
     "lerna": "^3.13.1",
+    "madr": "^2.1.2",
```

Comment:
Curious about these new deps, what do they do? Is there any scripting we should also be throwing up that uses them, to e.g. build ADR docs or whatever?

Reply:
Publishing ADR docs as HTML are a separate thing - I should probably file a follow-up issue for that. I think the upstream project intends that to be done with Jekyll & Github Pages, or whatever else we want to use for publishing docs.

[`madr`](https://github.com/adr/madr) is basically just the upstream source of templates and initial ADR.

[`adr-log`](https://github.com/adr/adr-log) is a quick utility useful for updating index.md when a new ADR is added.

Reply:
Sounds good. Fwiw, we are deploying some other docs to GH Pages already, so shouldn't be too much hassle to add these to the same build step in CI when we want to do that.